dev/ts/component/DropFileUserInterface.ts

```diff
@@ -25,7 +25,7 @@ export class DropFileUserInterface {
     private window: Window,
     private outerContainerId: string,
     private containerToFade: HTMLElement,
-    private dropCallback: (file: File) => void,
+    private dropCallback: (file: File) => Promise<void>,
```

Comment:
**thought:** I don't understand why we need this change. I will check locally.
I thought we managed all "promises" related problems.

Reply:
same answer than https://github.com/process-analytics/bpmn-visualization-js/pull/2824#discussion_r1338285156